releases/release-1.23/release-notes/maps/pr-103095-map.yaml

```diff
@@ -0,0 +1,4 @@
+pr: 103095
+releasenote:
+  text: Add support for `PodAndContainerStatsFromCRI` featuregate, which allows a
```

Comment:
Should it be feature gate?

Reply:
Yes, please see the discussion around this:
https://groups.google.com/g/kubernetes-sig-architecture/c/Nxsc7pfe5rw/m/vF2djJh0BAAJ
This discussion is in a file modified by this PR
https://github.com/kubernetes/kubernetes/pull/103095/files